examples/rbac/nginx/nginx-ingress-controller-rbac.yml

```diff
@@ -0,0 +1,111 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: nginx-ingress
+---  
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: nginx-ingress-serviceaccount
+  namespace: nginx-ingress
+---  
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: nginx-ingress-clusterrole
+rules:
+  - apiGroups:
+      - ""
+      - "extensions"
+    resources:
+      - configmaps
+      - secrets
+      - endpoints
+      - nodes
+      - pods
+    verbs:
+      - list
+      - watch
+  - apiGroups:
+      - ""
+    resources:
+      - services
+    verbs:
+      - get
+      - list
+      - watch
+      - update
```

Comment:
i think use does't to create role , you can only create serviceAccount , culster-role, rolebinding , it 's sample and correct.